examples/llama2/requirements-qlora.txt

```diff
@@ -1,6 +1,7 @@
 -r requirements.txt
 accelerate
 bitsandbytes
+git+https://github.com/jambayk/optimum.git@jambayk/constant-inputs
```

Comment:
would this branch merge into HF optimum?

Reply:
Our end goal is this method be the standard way to use different adapters in onnx models. So, I think we will probably be able to contribute this to optimum once the solution matures.

Reply:
if the private changes is not in the optimum, would this feature fail?

Reply:
Without these changes in my branch, we would not be able to use `ORTModelForCausalLM` for generation using this model. 
However, we are in the early stages of lora adapters deployment so the support for generation will be made available publicly (whether it is through optimum, onnxruntime genai library or custom model wrapper).

Reply:
thanks for the clarification.